website/templates/project/register_draft.mako

```diff
@@ -0,0 +1,60 @@
+<%inherit file="project/project_base.mako"/>
+
+<%def name="title()">Register ${node['title']}</%def>
+
+<%def name="content()">
+<div class="col-md-6 col-md-offset-3" id="draftRegistrationScope">        
+      <div id="embargo-addon" data-bind="with: $root.embargoAddon">
+        <hr />
+        
+        <p class="help-block">${language.REGISTRATION_EMBARGO_INFO}</p>
```

Comment:
Aside from using the `n` filter, is there a way to have line breaks when importing these strings from language.py? There is lot of html used throughout it to break up/format things as many of these text blobs are big.